src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -158,9 +168,47 @@ public String getDetailsPage(@PathVariable Long submissionId, Model model, final
 		model.addAttribute("analysisType", analysisType);
 		model.addAttribute("mailConfigured", emailController.isMailConfigured());
 
+		if(submission.getAnalysisState()==AnalysisState.ERROR) {
```

Comment:
Your formatting is off for this (no spaces around the `==`), can you fix it please.

Reply:
Fixed in 8448f0a75c32d57c156973710e4346c707a8567d